src/wmnds/components/content-tiles/related-content/_related-content.scss

```
@@ -0,0 +1,13 @@
+.wmnds-content-tile {
+  &--related-content {
+    h2 {
```

This should be a class like
```suggestion
    &__title {
```

instead of targeting a h2 element.